draft/spec/index.html

```diff
@@ -704,9 +704,13 @@ <h2>Inventory Digest</h2>
     <section id="version-inventory">
         <h2>Version Inventory and Inventory Digest</h2>
         <p>
-            Every version directory SHOULD include an inventory file that is an <a href="#inventory">Inventory</a>
-            of all content for versions up to and including that particular version. Every inventory file MUST have
-            a corresponding <a href="#inventory-digest">Inventory Digest</a>.
+            Every OCFL Object MUST have an  inventory file within the OCFL Object Root,
```

Comment:
Minor: extraneous space in `have an  inventory file`.
Otherwise: :+1:

Reply:
What @awoods said...